Polyfills/Console/Source/Console.h

```diff
@@ -16,13 +16,13 @@ namespace Babylon::Polyfills::Internal
         explicit Console(const Napi::CallbackInfo& info);
 
     private:
-        void Log(const Napi::CallbackInfo& info);
-        void Warn(const Napi::CallbackInfo& info);
-        void Error(const Napi::CallbackInfo& info);
-        void InvokeCallback(const Napi::CallbackInfo& info, Babylon::Polyfills::Console::LogLevel logLevel) const;
-        void InvokeEngineCallback(const std::string functionName, const Napi::CallbackInfo& info);
+        static void Log(const Napi::CallbackInfo& info);
+        static void Warn(const Napi::CallbackInfo& info);
+        static void Error(const Napi::CallbackInfo& info);
+        static void InvokeCallback(const Napi::CallbackInfo& info, Babylon::Polyfills::Console::LogLevel logLevel);
+        static void InvokeEngineCallback(const std::string functionName, const Napi::CallbackInfo& info);
 
-        std::unique_ptr<Napi::ObjectReference> m_engineConsole{};
-        Babylon::Polyfills::Console::CallbackT m_callback{};
+        static inline std::unique_ptr<Napi::ObjectReference> s_engineConsole{};
```

Comment:
I think this will cause an issue with shutdown? Since this is static, it will never be released right? I wonder if we can do something with the prototype chain instead of this.

Reply:
`SuppressDestruct` is no-op for JSI. Is it smth that can be fixed?
Other solution is to use a raw pointer allocation and never delete it.

Reply:
Prototype chain doesn't work because the functions are static🤦. But, I still managed to [implement a version](https://github.com/BabylonJS/BabylonNative/pull/1140) that doesn't use static globals. Thoughts?